extensions/uuid-generator/src/parseTypeID.ts

```diff
@@ -0,0 +1,23 @@
+import { Clipboard, getPreferenceValues, LaunchProps, showHUD, showToast, Toast } from "@raycast/api";
+import { fromString, toUUID } from "typeid-js";
+
+export default async (props: LaunchProps<{ arguments: { typeID: string } }>) => {
+  const { typeID } = props.arguments;
```

Comment:
The `typeID` is possibly `undefined` when calling this command through Deeplinks. 

For example:

```shell
open 'raycast://extensions/jmaeso/uuid-generator/parseTypeID'
```

Your command may crash in this step. And the author will receive some useless error reports in the dashboard. We'd better add a condition here:

```
if (!typeID) return;
```